src/App.js

```diff
@@ -6,6 +6,7 @@ import ResultHeader from './components/results/ResultHeader';
 import ResultList from './components/results/ResultList';
 import Footer from './components/footer/Footer';
 import Header from './components/header/Header';
+import Share from './components/share';
```

Comment:
Components filename should be named as the component, starting with capital letter -> ./components/share/Share.js